src/main/java/emissary/core/MobileAgent.java

```diff
@@ -40,7 +40,7 @@ public abstract class MobileAgent implements IMobileAgent, MobileAgentMBean {
     protected transient Thread thread = null;
 
     // Name for our threads
-    public static final String AGENT_THREAD = "MobileAgent-".intern();
```

Comment:
String literals are always interned, so these are redundant calls.  (Not technically within the scope of the original request, but these aren't numerous).